src/ACEtk/details/verify/src/positive.hpp

```diff
@@ -0,0 +1,29 @@
+template< typename Range, 
+          utility::Require< true, utility::is_range, Range > = true >
+inline bool positive( const Range& array ){
+  auto firstNegative = ranges::count_if( array, 
+    [](auto& e ){ return e < 0.0; } );
```

Comment:
you've confused positive and strictly positive

Reply:
You and I have a disagreement here. Stop by and let's chat about it.